optuna/integration/tensorflow.py

```diff
@@ -20,23 +20,9 @@ class TensorFlowPruningHook(SessionRunHook):
 
     Example:
 
-        Add a pruning SessionRunHook for a TensorFlow's Estimator.
-
-        .. code::
-
-                pruning_hook = TensorFlowPruningHook(
-                    trial=trial,
-                    estimator=clf,
-                    metric="accuracy",
-                    is_higher_better=True,
-                    run_every_steps=10,
-                )
-                hooks = [pruning_hook]
-                tf.estimator.train_and_evaluate(
-                    clf,
-                    tf.estimator.TrainSpec(input_fn=train_input_fn, max_steps=500, hooks=hooks),
-                    eval_spec
-                )
+        See example on GitHub for adding a pruning SessionRunHook for a TensorFlow's Estimator.
+        https://github.com/optuna/optuna/blob/master/examples/tensorflow_estimator_simple.py
```

Comment:
How about using the link instead of showing the URL?

![image](https://user-images.githubusercontent.com/3255979/76960510-f7980980-695e-11ea-9edc-23dccf8d4762.png)

Integration modules for XGBoost and Scikit-Optimize employ the links, so we can follow the convention.

![image](https://user-images.githubusercontent.com/3255979/76960533-02eb3500-695f-11ea-8821-7dca8702a556.png)

![image](https://user-images.githubusercontent.com/3255979/76960585-18605f00-695f-11ea-9740-95c2bbcda636.png)


And also, I think we can simplify the sentence because the words like TensorFlow and SessionRunHook are shown in the previous sentence.

The following is just an example:

> See [an example on GitHub](https://github.com/optuna/optuna/blob/master/examples/tensorflow_estimator_simple.py) to check the usage of the pruning hook.

Reply:
I think it would be good to include the commit hash in the link since
a path to an example may change in the future.
(e.g. `https://github.com/optuna/optuna/blob/30dba8814d332ff40d9c7465ee660a8ecf499547/examples/tensorflow_estimator_simple.py`)

For me, it also looks good not to simplify the words. 👍 
If we hold these words, it's reasonable for me to avoid repeatedly use `for`.
Let me put an example as well:

> See [the example](https://github.com/optuna/optuna/blob/30dba8814d332ff40d9c7465ee660a8ecf499547/examples/tensorflow_estimator_simple.py) if you want to add a pruning SessionRunHook for TensorFlow's Estimator.